health/health.go

```diff
@@ -0,0 +1,30 @@
+// health is depended on the code generated by protobuf
```

Comment:
// Package health provides some utility functions to health-check a server. The implementation // is based on protobuf. Users need to write their own implementations if other IDLs are used.